bob.js

```diff
@@ -1098,41 +1108,45 @@ username = (message) => {
 
 // Gets a formatted string for @ing a user
 mention = (user) => {
-    return "<@" + user.id + ">"; 
+    return "<@" + user.id + ">";
 }
 
 // Formats a time in seconds in H:mm:ss
-formatTime = (time) => {
+formatTime = (time, shorten) => {
     var hrs = Math.floor(time / 3600);
     var min = Math.floor((time - (hrs * 3600)) / 60);
-    var sec = time - (hrs * 3600) - (min * 60);
-    sec = Math.round(sec * 100) / 100;
+    var sec = Math.round((time - (hrs * 3600) - (min * 60)) * 100) / 100;
 
-    var result = (hrs < 10 ? "0" + hrs : hrs);
+    var result = hrs.toString();
     result += ":" + (min < 10 ? "0" + min : min);
     result += ":" + (sec < 10 ? "0" + sec : sec);
+    if (sec % 1 == 0) {
+        result += ".00";
+    } else if ((sec * 10) % 1 == 0) {
+        result += "0";
+    }
+    if (shorten != false) {
```

Comment:
Could just use `if (shorten) {`

Reply:
But sometimes shorten is undefined and then it should be treated as if it was true, I forgot to add a comment